pages/linux/tracepath.md

```diff
@@ -0,0 +1,28 @@
+# tracepath
+
+> Trace path to a network host discovering MTU along this path.
+> More information: <https://manned.org/tracepath>.
+
+- Trace the path to a host:
+
+`tracepath {{host}}`
+
+- Print numerical IP addresses only:
+
+`tracepath -n {{host}}`
+
+- Print both hostnames and numerical IP addresses:
+
+`tracepath -b {{host}}`
+
+- Specify maximum TTL (number of hops):
+
+`tracepath -m {{max_hops}} {{host}}`
+
+- Specify the initial destination port:
```

Comment:
Is it initial, because the destination port can change?

Reply:
Good question. I am not sure at all. It say initial everywhere I look. Should it be tested?

Reply:
If it's convenient for you @peterbabic then please do.  Otherwise, don't go out of your way—this is what the man page says and if it needs clarification someone can come along and fix it.  We frequently get PRs to clarify this or that 👍🏻

Reply:
My theory is, that for example if you send a request to a https port, but the destination has http only, it gets redirected, so that the destination port isn't the intended port

Reply:
Sounds plausible, I try to test this in the morning.

Reply:
Option p with tracepath command will allow to set destination port to be used. Whenever we would need to set a fixed port at the destination probe, we can use this option. It generally limits the usage of the port options.

Reply:
- `tracepath peterbabic.dev` returns `no reply Too many hops` after TTL is reached
- `traceroute peterbabic.dev` returns the path
- `mtr -w peterbabic.dev` return the path

Looking at https://man.archlinux.org/man/core/traceroute/traceroute.8.en#default it provides a hint.
Consequentially:

- `tracepath -p 33434 peterbabic.dev` returns the path

The host probed in the examples contains post with further analysis.

What to do with this information?

Reply:
I have added a new commit, suggesting a an initial port related change (I did not find how can I add a review like you all do).

Reply:
>What to do with this information?

Great, thank you @Waples and @peterbabic for your work.  I'm not sure what the best thing to do here is, but probably what I'd do for now it leave it.  It's fine as-is, I think, and is at least as good as the man page.  @peterbabic I think your commit is right, but I have a comment about the formatting of that information.

>I did not find how can I add a review like you all do

It's really simple: (1) go to the "Files Changed" tab, (2) hover over a line, (3) click (and drag, if you want) the blue plus icon, (4) type your comment.